site/src/HomePage/HomePage.tsx

```diff
@@ -187,7 +187,7 @@ export const HomePage = () => {
                 </Box>
                 <Code
                   language="tsx"
-                  errorTokens={['brandd', 'large']}
+                  errorTokens={['brand_', 'large']}
```

Comment:
I presume that `brand_` is just as good for highlighting that.

That said, if you want to keep these spellings, I'm happy to drop these changes.

Reply:
I get they are both incorrect values yeah. Personally though i'd prefer to stick the the "typo" that's in there.

Reply:
Dropped